hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaDeploymentManagement.java

```diff
@@ -342,6 +352,12 @@ private void assertMaxTargetsPerManualAssignmentQuota(final Long distributionSet
 
     private Set<Long> closeOrCancelActiveActions(final AbstractDsAssignmentStrategy assignmentStrategy,
             final List<List<Long>> targetIdsChunks) {
+
+        if (isMultiAssignmentsEnabled()) {
+            LOG.debug("Multi Assignments feature is enabled: No need to close /cancel active actions.");
+            return Collections.emptySet();
```

Comment:
Return type `Set<Long>` is not used anymore, so I think it could be removed and changed to `void` instead